addons/website_livechat/static/tests/tours/website_livechat_chatbot_continue.js

```diff
@@ -8,7 +8,7 @@ registry.category("web_tour.tours").add("website_livechat.chatbot_continue_tour"
             trigger: messagesContain("Hello, what can I do for you?"),
         },
         {
-            trigger: ".o-livechat-root:shadow li:contains(No, thank you for your time.)",
+            trigger: ".o-livechat-root:shadow button:contains(No, thank you for your time.)",
```

Comment:
`:contains` didn't work for finding the text in child element?

Reply:
`:contains` work, but the `click` action on the `li` element has no effect, hence the change.